addons/packages/kapp-controller/0.30.0/bundle/config/schema.yaml

```diff
@@ -0,0 +1,45 @@
+#! schema.yaml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for kapp-controller"
+---
+#@schema/desc "The namespace value used by older templates, will overwrite with top level namespace of present, for backward compatibility"
+namespace: kapp-controller
+#@schema/desc "Configuration for kapp-controller"
+kappController:
+  #@schema/desc "The namespace value used by older templates, will overwrite with top level namespace of present, for backward compatibility"
+  #@schema/nullable
+  namespace: kapp-controller
+  #@schema/desc "Whether to create namespace specified for kapp-controller"
+  createNamespace: true
+  #@schema/desc "The namespace value used for global packaging resources. Any Package and PackageMetadata CRs within that namespace will be included in all other namespaces on the cluster, without duplicating them"
+  globalNamespace: tanzu-package-repo-global
+  deployment:
+    #@schema/desc "The coreDNSIP will be injected into /etc/resolv.conf of kapp-controller pod"
+    #@schema/nullable
+    coreDNSIP: ""
+    #@schema/desc "HostNetwork of kapp-controller deployment"
+    #@schema/nullable
+    hostNetwork: ""
+    #@schema/desc "PriorityClassName of kapp-controller deployment"
```

Comment:
Do such fields such as "hostNetwork" or "hostNetwork" or alike, need to be more descriptive, e.g. explaining in high level, what is "PriorityClassName"?

Reply:
For priorityClassName, I can add additional line => "This value references an existing Kubernetes priorityClassName that can be assigned to kapp-controller pods"
What do you think?

Reply:
Yes sounds good, or the following from kapp controller in [here](https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/vendor/k8s.io/api/core/v1/types.go): "The priority value that various system components use to find the priority of the pod"

Reply:
Updated the descriptions